pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -63,11 +72,15 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
   applications,
   onClose,
 }: IAnalysisWizard) => {
-  const title = "Application analysis";
-  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!isInitTasks) {
+      initTasks();
+    }
 
-  const [isInitTasks, setInitTasks] = React.useState(false);
-  const [createdTasks, setCreatedTasks] = React.useState<Array<Task>>([]);
+    return () => {
+      setInitTasks(false);
+    };
+  }, []);
```

Comment:
I don't see why we need useEffect (I actually initially tried that path but came back from it).
I React state isInitTasks allows us to flag if the task(s) have been created and if not then this is done only once.

I actually get an error on here: React Hook useEffect has missing dependencies: 'initTasks' and 'isInitTasks'. Either include them or remove the dependency array.

Reply:
Ok I will revert the change